pages/users/signin.tsx

```diff
@@ -0,0 +1,136 @@
+import { signIn } from 'next-auth/client';
+import { Grid, TextField, Button } from '@material-ui/core';
+import { Formik, Form } from 'formik';
+import SigninSchema from 'interfaces/signin';
+import Layout from 'components/Layout';
+import styles from '../../styles/users/Signin.module.css';
+
+interface FormValues {
+  email: string;
+  password: string;
+}
+
+interface ErrorValues {
+  email?: string;
+  password?: string;
+}
+
+// UserSignIn page. Will need additional email verification to be able to create organizations.
+const UserSignIn: React.FC = () => {
+  const handleSubmit = async (values: FormValues): Promise<any> => {
+    // Sign in with credentials
+    try {
+      signIn('credentials', {
+        email: values.email,
+        password: values.password,
+        callbackUrl: 'http://localhost:3000',
+      });
+    } catch {
+      throw new Error('Could not sign in.');
+    }
+  };
+
+  // Validates inputs
+  const validate = (values: FormValues): ErrorValues => {
+    const errors: { [k: string]: string } = {};
+    const { error } = SigninSchema.validate(values, {
+      context: { strict: true },
+      abortEarly: false,
+    });
+
+    // Fill out the errors payload with the correct messaging
+    if (error) {
+      for (let i = 0; i < error.details.length; i += 1) {
+        const val = error.details[i];
+        errors[val.path[0]] = val.message;
+      }
+    }
+    return errors;
+  };
+
+  const constructRow = (
+    varName: string,
+    handleChange: any,
```

Comment:
handleChange type should be `FormikHandlers['handleChange']` imported from Formik